### ISSUE TITLE
Auto-scaling for kube-flagd-proxy based on metrics (nice to have)

### ISSUE THREAD

**Issue by toddbaert** [open-feature/open-feature-operator#379]
Once https://github.com/open-feature/flagd/issues/482 is done, we could have intelligent auto-scaling for the flagd-kube-proxy based on metrics related to load.

**Comment by beeme1mr** [comment]
Closing due to inactivity. Please open a new issue and reference this one if this functionality is required.
